lib/dockerregistry/checks_test.go

```diff
@@ -317,3 +320,60 @@ func TestImageSizeCheck(t *testing.T) {
 			fmt.Sprintf("checkError: test: %v (ImageSizeCheck)\n", test.name))
 	}
 }
+
+func TestImageVulnCheck(t *testing.T) {
+	mkPopulateRequestsFake := func(
+		vulnerabilities []*grafeaspb.VulnerabilityOccurrence,
```

Comment:
You should probably use a mapping instead of a flat list. That is, something like `map[Digest]*grafeaspb.VulnerabilityOccurrence`.

Reply:
I still think using Digest's wouldn't add anything to the tests. Each unit test is just running through the vulnerabilities for one image.